code/modules/research/design_database_analyzer.dm

```diff
@@ -71,8 +69,17 @@
 				cached_materials -= mat
 		return TRUE
 
+	if(!(stat & (BROKEN|NOPOWER)))
+		interface_interact(user)
+		return TRUE
```

Comment:
This shouldn't be called from here.

Reply:
???

Reply:
Why can't I open the network_device UI from attack_hand?

Reply:
Because it's redundant with `interface_interact` below, except that it runs the wrong checks.

Reply:
weird, but that's the only way it works?

Reply:
Testing, give me a minute.

Reply:
you're right, it works. Pushed.